system/index.js

```diff
@@ -66,7 +73,7 @@ module.exports = async (message) => {
         throw new Error('Invalid return type, middleware must return an array of arguments')
       }
     } catch (error) {
-      logger.error({error});
+      console.error(error);
```

Comment:
I don't love this mostly because it means we no longer get structured logs for these type of failures. I wonder if `logger.error(error.toString())` or `logger.error({error: error.toString()})` is an OK replacement so we can keep structured logs.

Reply:
> I don't love this mostly because it means we no longer get structured logs for these type of failures. I wonder if `logger.error(error.toString())` or `logger.error({error: error.toString()})` is an OK replacement so we can keep structured logs.

I know what you mean about the structured logging.   I'm wanting to avoid even more dependency on it until we decide on whether it's the way forward.  I would be more tempted to use `Winston` instead.  Let me see what kind of output I get with the `.toString()`. approach.

Reply:
```
 ›   Error: {"name":"Evergreen Logger","hostname":"358a826296e6","pid":1,"level":50,"msg":"Could not locate user function at /workspace: Error: Cannot find module 'sequelize'\nRequire stack:\n-
 ›   /workspace/square.js\n- /layers/heroku_node-function/system/index.js\n- /layers/heroku_node-function/riff-invoker-node/server.js","time":"2020-03-06T23:46:34.707Z","v":0}
 ›   Error: Could not locate user function at /workspace: Error: Cannot find module 'sequelize'
 ›   Require stack:
 ›   - /workspace/square.js
 ›   - /layers/heroku_node-function/system/index.js
 ›   - /layers/heroku_node-function/riff-invoker-node/server.js
 ›   Error: {"name":"Evergreen Logger","hostname":"358a826296e6","pid":1,"level":60,"msg":"Could not locate user function at /workspace: Error: Cannot find module 'sequelize'\nRequire stack:\n-
 ›   /workspace/square.js\n- /layers/heroku_node-function/system/index.js\n- /layers/heroku_node-function/riff-invoker-node/server.js","time":"2020-03-06T23:46:34.710Z","v":0}
 ›   Error: Context Done: Killing Container 358a826296e62f11c313b52eca222af17612c1deceaecd3c2a1ca7af3908b9cb
 ›   Error: Error response from daemon: Cannot kill container: 358a826296e62f11c313b52eca222af17612c1deceaecd3c2a1ca7af3908b9cb: Container 358a826296e62f11c313b52eca222af17612c1deceaecd3c2a1ca7af3908b9cb
 ›   is not running
```